mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -787,15 +787,15 @@ protected void modSeqGreaterThanShouldReturnUidsOfMessageHavingAGreaterModSeq()
         SearchQuery searchQuery = SearchQuery.of(SearchQuery.modSeqGreaterThan(7L));
 
         assertThat(messageSearchIndex.search(session, mailbox, searchQuery))
-            .containsOnly(m7.getUid(), m8.getUid(), m9.getUid());
+            .containsOnly(m8.getUid(), m9.getUid());
```

Comment:
Changes to AbstractMessageSearchIndexTest class needs to go in the same commit than  'gte,lte' -> 'lt,gt' stuff